newdle/api.py

```diff
@@ -355,6 +357,21 @@ def update_participant(args, code, participant_code):
     for key, value in args.items():
         setattr(participant, key, value)
     db.session.commit()
+    if participant.newdle.notify:
+        notify_newdle_creator(
+            participant.newdle,
+            participant,
+            f'{participant.name} joined to {participant.newdle.title}',
```

Comment:
"joined to" sounds rather strange. What about "{name} replied to {title}" or something like that?